Src/WitsmlExplorer.Frontend/components/Modals/LogPropertiesModal.tsx

```diff
@@ -42,7 +42,7 @@ const LogPropertiesModal = (props: LogPropertiesModalInterface): React.ReactElem
   };
 
   const onChangeCurve = async (event: any) => {
-    const indexCurve = event.target.value === IndexCurve.Time ? IndexCurve.Time : IndexCurve.Depth;
+    const indexCurve = event[0] === IndexCurve.Time ? IndexCurve.Time : IndexCurve.Depth;
```

Comment:
Should be `event.selectedItems[0]` not `event[0]`

Reply:
Fixed